### ISSUE TITLE
Payment option: Zcash

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#762]
### Problem

Bitcoin has many advantages, the biggest on is Lightning Network, which can scale up to arbitrary amounts, but has some disadvantages, especially when it comes to privacy. For some users this might be a deal-breaker.

### Solution

Zcash might be an interesting payment option to support, especially if it's done via a [light-wallet](https://github.com/adityapk00/zecwallet-lite).

**Comment by sandrina-p** [comment]
A similar argument can be used to support other methods outside of the cryptocurrency world:

> **Problem** Cryptocurrency has many advantages, the biggest on is privacy, but has some disadvantages, especially when it comes to ease of use and popularity. For some users, this might be a deal-breaker.
> **Solution:** Paypal or Strype might be an interesting payment option to support.

**Comment by taoeffect** [comment]
Yes @sandrina-p, feel free to take charge and create the PayPal payment option issue. Let's leave this issue for Zcash-related discussion.

**Comment by sandrina-p** [comment]
@taoeffect Done at #770
